x-pack/plugins/security_solution/public/management/cypress/e2e/response_actions/response_console/file_operations.cy.ts

```diff
@@ -21,8 +21,7 @@ import { enableAllPolicyProtections } from '../../../tasks/endpoint_policy';
 import { createEndpointHost } from '../../../tasks/create_endpoint_host';
 import { deleteAllLoadedEndpointData } from '../../../tasks/delete_all_endpoint_data';
 
-// Failing: See https://github.com/elastic/kibana/issues/170424
-describe.skip('Response console', { tags: ['@ess', '@serverless'] }, () => {
+describe('Response console', { tags: ['@ess', '@serverless'] }, () => {
```

Comment:
You can also undo the change on line 115 on this file. Since Kibana `v18.2` is on node `18.18.2`, the assumption that the package unzip causes issues on the test is not relevant on this branch. So it should be back to `expect(unzippedFileContent).to.equal(fileContent);`

Reply:
done: https://github.com/elastic/kibana/pull/175780/commits/ad9c08303bef76941df1aa91614cd1fc11820ea0

Reply:
Thanks for the changes 🚀